src/NetworkEquipment.php

```diff
@@ -302,6 +319,15 @@ public function rawSearchOptions()
             'datatype'           => 'dropdown'
         ];
 
+        $tab[] = [
+            'id'                 => '10',
+            'table'              => $this->getTable(),
+            'field'              => 'remote_addr',
+            'name'               => __('Public contact address'),
```

Comment:
I don't think this is how we want to name it for network devices. For computers, this is the "remote_addr" of the agent which submitted an inventory. So the name is correct in that case as the agent contact GLPI from that ip. For network devices, this is really the ip on which SNMP requests have to be made. Something like "SNMP port remote address" would be more appropriate.

Reply:
done